server/api/db/sqldb/db.py

```diff
@@ -2605,6 +2612,7 @@ def get_project_summary(
                 f"Project summary not found: project={project}"
```

Comment:
not related but nice python feature can be used here
```suggestion
                f"Project summary not found: {project=}"
```